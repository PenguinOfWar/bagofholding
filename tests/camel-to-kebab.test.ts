import camelToKebab from '@/packages/camel-to-kebab/camel-to-kebab';

it('shows `camelKebabsAnybody` as `camel-kebabs-anybody`', () => {
  expect(camelToKebab('camelKebabsAnybody')).toBe('camel-kebabs-anybody');
  expect(camelToKebab('camelKebabsAnybody')).toMatchSnapshot();
});
