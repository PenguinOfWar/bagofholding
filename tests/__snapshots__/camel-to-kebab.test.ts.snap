// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`shows \`camelKebabsAnybody\` as \`camel-kebabs-anybody\` 1`] = `"camel-kebabs-anybody"`;
